[2024-04-05 19:36:18,067] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-04-05 19:36:18,072] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-04-05 19:36:18,072] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-04-05 19:36:18,072] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-04-05 19:36:18,072] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-04-05 19:36:18,092] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2024-01-01 00:00:00+00:00
[2024-04-05 19:36:18,095] {standard_task_runner.py:52} INFO - Started process 725881 to run task
[2024-04-05 19:36:18,098] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/extrai_dados_climaticos.py', '--cfg-path', '/tmp/tmptogm1_dw', '--error-file', '/tmp/tmppxqo547y']
[2024-04-05 19:36:18,099] {standard_task_runner.py:80} INFO - Job 106: Subtask extrai_dados
[2024-04-05 19:36:18,133] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-01-01T00:00:00+00:00 [running]> on host TEC017860.
[2024-04-05 19:36:18,181] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T00:00:00+00:00
[2024-04-05 19:36:18,849] {python.py:173} INFO - Done. Returned value was: None
[2024-04-05 19:36:18,854] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20240101T000000, start_date=20240405T223618, end_date=20240405T223618
[2024-04-05 19:36:18,892] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-05 19:36:18,903] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
